### PR TITLE
Update an account mode from cli

### PR DESCRIPTION
**Pull request by privateducky**

### CODE REVIEW

File: python/sdk/prelude_sdk/controllers/iam_controller.py [hunk 1]
```diff
@@ -32,6 +32,15 @@ def purge_account(self):
                 raise Exception(res.text)
             return res.text
 
+    @verify_credentials
+    def update_account(self, mode: int):
+        """ Update properties on an account """
+        with Spinner():
+            params = dict(mode=mode)
+            res = requests.put(f'{self.account.hq}/iam/control', headers=self.account.headers, json=params)
```
Comment (makk94) [inline python/sdk/prelude_sdk/controllers/iam_controller.py:40]: /iam/account

File: python/cli/prelude_cli/views/iam.py [hunk 2]
```diff
@@ -25,6 +25,18 @@ def register_account(controller):
     print_json(data=creds)
 
 
+@iam.command('update-account')
+@click.option('-m', '--mode',
+              help='provide a mode',
+              default='manual', show_default=True,
+              type=click.Choice(['manual', 'frozen'], case_sensitive=False))
```
Comment (makk94) [inline python/cli/prelude_cli/views/iam.py:32]: if we might add more modes, I'd dynamically create this list from the enum like we do elsewhere

File: python/cli/prelude_cli/views/iam.py [hunk 3]
```diff
@@ -25,6 +25,18 @@ def register_account(controller):
...
+@click.pass_obj
+@handle_api_error
+def update_account(controller, mode):
+    """ Update an account """
+    controller.update_account(mode=mode)
```
Comment (makk94) [inline python/cli/prelude_cli/views/iam.py:37]: sdk param is listed as an int, what's this?

File: python/cli/prelude_cli/views/iam.py (hunk 2 above)
Comment (privateducky) [reply python/cli/prelude_cli/views/iam.py:32]: Agreed

File: python/sdk/prelude_sdk/controllers/iam_controller.py (hunk 1 above)
Comment (privateducky) [reply python/sdk/prelude_sdk/controllers/iam_controller.py:40]: Good eye

File: python/cli/prelude_cli/views/iam.py (hunk 3 above)
Comment (privateducky) [reply python/cli/prelude_cli/views/iam.py:37]: That's me flipping between a string, an enum and an int, followed by you having a better eye than me!

**Pull request merged**